tests/periph_timer_periodic/tests/01-run.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2020 Benjamin Valentin <benpicco@beuth-hochschule.de>
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import sys
+import time
+from testrunner import run
+
+
+def testfunc(child):
+    child.expect_exact('TEST START')
+    start = time.time()
+    child.expect_exact('TEST SUCCEEDED')
+    end = time.time()
+    assert (end - start) > 5
+    assert (end - start) < 6
```

Comment:
Needs to be updated to new value of CYCLE_MS. Maybe it makes sense to add a comment here, in case CYCLE_MS is ever touched again?

Reply:
I had another idea: Adjusted the number of timer iterations so that we always get a printout every second / a run-time of 5s.

Reply:
Is there a reason to run the test that long? I mean, couldn't it finish in one second, doing 200ms steps?

Reply:
No, I only made it so that it's easy for humans to recognize the pattern.
But since we have automated testing, there is no need to accommodate for such peculiarities.

I'll make it go faster then to speed up automatic tests.

Reply:
`make test` works again (at least on Arduino Mega2560 and MSB-A2) and is completed in ~1 sec